### ISSUE TITLE
zap install krita / this app does not provide any candidate for installation

### ISSUE THREAD

**Issue by Semnodime** [srevinsaju/zap#62]
`zap search krita`
```
Krita by 
<p>Krita is the full-featured digital art studio.</p>
<p>It is perfect for sketching and painting, and presents an end–to–end solution for creating digital painting files from scratch by masters.</p>
<p>Krita is a great choice for creating concept art, comics, textures for rendering and matte paintings. Krita supports many colorspaces like RGB and CMYK
      at 8 and 16 bits integer channels, as well as 16 and 32 bits floating point channels.</p>
<p>Have fun painting with the advanced brush engines, amazing filters and many handy features that make Krita enormously productive.</p>

Install it by
zap install krita
```
`zap install krita`
```
[FATAL] 2022/01/20 19:54:47 main.main:main.go:163 this app does not provide any candidate for installation
```

**Comment by srevinsaju** [comment]
Agreed, unfortunately, the Krita appimage.github.io page has no Download button, and hence we are not able to access them: https://appimage.github.io/Krita/

**Comment by Semnodime** [comment]
So, if I understand correctly, `zap` requires the Download-Button on the `appimage.github.io` page.
I want to kindly suggest printing some error message which clarifies why no candidate is available (maybe along the `appimage.github.io` link, so the user can download the appimage manually).

In the specific case of `krita`, the homepage under https://krita.org/en/download/krita-desktop/ provides the `.appimage`.
Also, [krita's github page](https://github.com/KDE/krita) lists the sources for both the latest stable and the unstable release:
- https://binary-factory.kde.org/job/Krita_Stable_Appimage_Build/
- https://binary-factory.kde.org/job/Krita_Nightly_Appimage_Build/

They can also be parsed from their respective zsync files, which appear to have a permanent link:
- https://binary-factory.kde.org/job/Krita_Stable_Appimage_Build/lastCompletedBuild/artifact/Krita-Plus-x86_64.appimage.zsync/*view*/
- https://binary-factory.kde.org/job/Krita_Nightly_Appimage_Build/lastCompletedBuild/artifact/Krita-Next-x86_64.appimage.zsync/*view*/

Is there some way to integrate it into zap?

**Comment by srevinsaju** [comment]
Yes, we should be able to do it if we are able to move to our own catalog of appimages. Currently, we depend on `appimage.github.io`, and appimage.github.io doesn't support these. Maybe we could support them, in a separate backend built separately by us. What do you think? I think this is a good idea on the long term :thinking:

**Comment by Rabioli** [comment]
Yes it would definitely be a good idea, I was just working in a shell script to hackily doing this, but if we can integrate this into a proper program that would be great.

We could have a parser for gitlab, github as mentioned in  #77, and other sources outside of the ones the program already has.

I would be willing to help since I have some time now but I wouldn't know where to start
